event/management/commands/instagram_backfill.py

```diff
@@ -0,0 +1,86 @@
+import requests
+import time
+import environ
+import random
+import re
+from scrapy.selector import Selector
+
+from django.core.management.base import BaseCommand
+
+from event.models import Artist, Instagram
+
+env = environ.Env()
+
+def parse_stat(formatted_stat):
+    if (not len(formatted_stat)):
+        return None
+
+    if (formatted_stat[-1] == "M"):
+        return int(formatted_stat[:-1].replace(',','')) * 1_000_000
+
+    if (formatted_stat[-1] == "K"):
+        return int(formatted_stat[:-1].replace(',','')) * 1_000
+
+    return int(formatted_stat.replace(',',''))
+    
+
+def get_instagram(url):
+    print(f"url: {url}")
+    response = requests.get(url)
+
+    if response.status_code != 200:
+        print(f"response: {response.status_code}")
+        print(f"error: {response.text}")
+        return -1
+
+    meta_description_content = Selector(text=response.text).xpath("//meta[@name='description']/@content")[0].extract()
+    stats_meta, handle_meta = meta_description_content.split("-")
+
+    stats_parts = stats_meta.split()
+    
+    followers=parse_stat(stats_parts[0])  
+    following=parse_stat(stats_parts[2])
+    posts=parse_stat(stats_parts[4])
+
+    handle_matches = re.findall(r"\((.*?)\)", handle_meta)
+    handle = handle_matches[0] if len(handle_matches) else ""
```

Comment:
typo: handler?

Reply:
I changed it to "handler" for consistency with the models and the twitter version of this, but I believe the used term is "handle". (https://whatagraph.com/blog/articles/social-media-handles) We could make change globally later if you think it makes sense.